README.md

```diff
@@ -1,87 +1,90 @@
-<br />
+# Fern
+
+![YC](https://img.shields.io/badge/Y%20Combinator-2023-orange)
+![License](https://img.shields.io/badge/License-MIT-blue)
+
+[![Discord](https://img.shields.io/badge/Join%20Our%20Community-black?logo=discord)](https://discord.com/invite/JkkXumPzcG)
+[![Documentation](https://img.shields.io/badge/Read%20our%20Documentation-black?logo=book)](https://www.buildwithfern.com/docs/intro)
 
-<div align="center">
-Backed by Y Combinator
 </div>
 
----
+Fern is an open source toolkit for designing, building and consuming REST APIs. With Fern, you can generate client libraries, documentation and boilerplate for your backend server in your chosen framework (e.g. FastAPI, Express).
+
+Fern can be used in your existing projects through OpenAPI or if you choose, you can describe your API in the Fern specification. In this sense, Fern is an alternative to the OpenAPI standard but does not aim to replace it.
+
+Fern takes care of the boilerplate associated with developing APIs, so you can **focus on delivering value**.
 
-**Fern is an open source format for defining REST APIs.**
-You can think of it like a programming language to describe
-your API: your endpoints, types, errors, and examples.
+# Capabilities
 
-This repository contains the **Fern compiler.** The compiler transforms the API description into useful outputs, like:
+Currently, the Fern compiler can generate the following types of artifacts:
 
-### 🌿 SDKs
+### 🌿 Client Libraries and SDKs
 
-Client libraries speed up internal development, and help acquire customers who use your API. Our auto-generated SDKs are idiomatic and feel handwritten.
+Whether you call them SDKs, client libraries, bindings, or wrappers, Fern generates idiomatic SDKs you can use to interact with your APIs, or publish for third party usage. This makes keeping your client libraries up to date with your API endpoints trivial, and speeds up your workflows dramatically.
 
-### 🌿 Server-side code generation
+Currently, the following languages are supported:
 
-We automatically generate lots of boilerplate on the server side, like Pydantic models for FastAPI and Jersey interfaces for Spring Boot. We also add compile-time validation that all your endpoints are being served correctly.
+- [TypeScript](https://github.com/fern-api/fern-typescript)
+- [Java](https://github.com/fern-api/fern-java)
+- [Python](https://github.com/fern-api/fern-python)
+- [Go](https://github.com/fern-api/fern-go)
 
-### 🌿 Postman Collection
+If you'd like to see a language supported, head over to [the Fern issues page](https://github.com/fern-api/fern/issues) and vote for the language you'd like to see supported next.
 
-Complete with examples of successful and unsuccessful requests!
+### 🌿 Backend Code
 
-### 🌿 An OpenAPI spec
+Define your API, and Fern will generate models, endpoints and boilerplate application code. All the generated code is validated at build time, ensuring your all your endpoints and their data are being served correctly.
```

Comment:
> Define your API, and Fern will generate models, endpoints and boilerplate application code.

Fern will generate the models, but only part of the endpoints. An endpoint is made up of networking logic + business logic and fern handles all the boilerplate networking logic like parsing path parameters, query parameters, sending errors.

Reply:
> All the generated code is validated at build time, ensuring your all your endpoints and their data are being served correctly.

Is "compile" time better than "build" time? Also the generated code is not "validated" but the endpoint implementation is validated. 

Maybe something like: 

The generated server code provides autocompletion and type-safety, ensuring that all your endpoints are served correctly and not diverging from the API definition. For example, if you return the wrong type for an endpoint, you’ll get a compile error.